### DISCUSSION TITLE
usin with angular material

### DISCUSSION THREAD

**Discussion by mgallardoDev** [MurhafSousli/ngx-scrollbar#401] [Q&A] [answered]
Hi! i have a problem making it work with material.

my scrollable component is not a div, is a mat-sidenav-content, if i wrap it qith de <ngscrollbar> it doesnt works

its necesary thar the container would be a div?

**Comment by MurhafSousli** [comment, marked as answer]
Just add `<ng-scrollbar>` inside it as part of its content
